week3project/main.js

```diff
@@ -0,0 +1,81 @@
+const wrapper = document.querySelector(".wrapper"),
+inputPart = wrapper.querySelector(".input-part"),
+inputfield = inputPart.querySelector("input"),
+infotxt = inputPart.querySelector(".info-txt"),
+icons = document.querySelector(".weather-part img"),
+backarrow = wrapper.querySelector(".h1 i"),
+searchhistory = wrapper.querySelector(".searches");
+let api;
+
+
+backarrow.addEventListener("click", function(){
+    wrapper.classList.remove("active");
+});
+
+inputfield.addEventListener("keyup", function(e){
+if(e.key === "Enter" && inputfield.value != ""){
+    requestApi(inputfield.value);       
+}
+setTimeout(() => {
+  if(e.key === "Enter" && inputfield.value != ""){
+       var paragraph = document.createElement("p");
+       paragraph.innerText = inputfield.value;
+        searchhistory.appendChild(paragraph);
+        
+    }
+ }, 3000);
+});
+
+var apiKey = 'a9bc860599772b42085366737fb5388d';
```

Comment:
Ideally, we don't want to put apiKeys in front end code. This is just a toy project so maybe it is not so important but keep a note of that